src/Infrastructure/Localization/Startup.cs

```diff
@@ -2,42 +2,40 @@
 using FSH.WebApi.Infrastructure.Middleware;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Localization;
+using Microsoft.AspNetCore.Mvc.Razor;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Localization;
+using OrchardCore.Localization;
 
 namespace FSH.WebApi.Infrastructure.Localization;
 
 internal static class Startup
 {
-    internal static IServiceCollection AddLocalization(this IServiceCollection services, IConfiguration config)
+    internal static IServiceCollection AddPOLocalization(this IServiceCollection services)
     {
-        services.AddLocalization();
+        services.AddMvc()
+        .AddViewLocalization(LanguageViewLocationExpanderFormat.Suffix);
```

Comment:
I don't think we need this... we don't have any mvc views in the app, do we?

Reply:
We use models and views in email template